subsys/mgmt/osdp/Kconfig

```diff
@@ -14,6 +14,19 @@ menuconfig OSDP
 
 if OSDP
 
+choice
+	prompt "OSDP Mode of Operation"
+	default OSDP_MODE_PD
```

Comment:
Default value will be systematically the choice proposition.
So we usually put propositions in such order that the first one is the default and don't use explicit 'default'.
Please reorder accordingly.

Reply:
Done. Thanks.